sopel/plugin.py

```diff
@@ -820,31 +823,168 @@ def add_attribute(function):
     return add_attribute
 
 
-def rate(user: int = 0, channel: int = 0, server: int = 0) -> typing.Callable:
+def rate(
+    user: int = 0,
+    channel: int = 0,
+    server: int = 0,
+    *, message: typing.Optional[str] = None,
```

Comment:
Looks weird. Not that the alternatives (`*,` at the end of the previous line, or on its own line) look great either, but let me try this on for size:

```suggestion
    *,
    message: typing.Optional[str] = None,
```

(Nope, it still looks weird. Damn it.)

Reply:
Ah ah I know right?